libsolutil/StringUtils.h

```diff
@@ -111,7 +111,7 @@ std::string joinHumanReadablePrefixed
 /// like  0x5555...{+56 more}...5555
 /// @example formatNumber((u256)0x7ffffff)
```

Comment:
I would suggest to keep only one documentation and just document what's different in the other function, e.g. here you could just say `/// Same as @ref formatNumberReadable but only for unsigned numbers`